omero/sysadmins/unix/index.txt

```diff
@@ -14,9 +14,19 @@ additional pages listed below.
 
     server-installation
     server-centos6
+    server-centos6-ice36
     server-centos7
     server-linux-walkthrough
     server-install-homebrew
     server-binary-repository
     server-postgresql
     install-web
+
+.. toctree::
+    :maxdepth: 2
+    :hidden:
+
+    walkthrough/requirements
+    walkthrough/requirements_centos6
```

Comment:
I mentioned it elsewhere as well, but can we please drop the whole requirements_centos6 and leave just the py27 versions?  No one doing a new install should be using python 2.6, so I don't think having this before the real versions people should be using in the docs serves any purpose other to confuse and misdirect.